src/metabase/task.clj

```diff
@@ -305,3 +305,14 @@
   []
   {:scheduler (some-> (scheduler) .getMetaData .getSummary str/split-lines)
    :jobs      (jobs-info)})
+
+(defmacro rerun-on-error
+  "Retry the current Job if an exception is thrown by the enclosed code."
+  {:style/indent 1}
```

Comment:
I added this metadata in desperation before editing the cljfmt edn file. Is it worth keeping? @camsaul